DevelopGitHub/GitHubExtension/GitHubExtension.DB/PullRequestTemplates.sql

```diff
@@ -0,0 +1,6 @@
+﻿CREATE TABLE [dbo].[PullRequestTemplates]
```

Comment:
There's no difference, but they contains different values. Is it better to create one table instead of two?